Source/Core/AudioCommon/CubebStream.cpp

```diff
@@ -61,43 +69,55 @@ bool CubebStream::Init()
   if (!m_coinit_success)
     return false;
   Common::Event sync_event;
-  m_work_queue.Push([this, &return_value, &sync_event] {
-    Common::ScopeGuard sync_event_guard([&sync_event] { sync_event.Set(); });
-#endif
-
-    m_ctx = CubebUtils::GetContext();
-    if (m_ctx)
+  m_work_queue.Push(
+    [this, &return_value, &sync_event]
     {
-      m_stereo = !Config::ShouldUseDPL2Decoder();
+      Common::ScopeGuard sync_event_guard([&sync_event] { sync_event.Set(); });
+#endif
 
-      cubeb_stream_params params{};
-      params.rate = m_mixer->GetSampleRate();
-      if (m_stereo)
+      m_ctx = CubebUtils::GetContext();
+      if (m_ctx)
       {
-        params.channels = 2;
-        params.format = CUBEB_SAMPLE_S16NE;
-        params.layout = CUBEB_LAYOUT_STEREO;
+        m_stereo = !Config::ShouldUseDPL2Decoder();
+
+        cubeb_stream_params params{};
+        params.rate = m_mixer->GetSampleRate();
+        if (m_stereo)
+        {
+          params.channels = 2;
+          params.format = CUBEB_SAMPLE_S16NE;
+          params.layout = CUBEB_LAYOUT_STEREO;
+        }
+        else
+        {
+          params.channels = 6;
+          params.format = CUBEB_SAMPLE_FLOAT32NE;
+          params.layout = CUBEB_LAYOUT_3F2_LFE;
+        }
+
+        u32 minimum_latency = 0;
+        if (cubeb_get_min_latency(m_ctx.get(), &params, &minimum_latency) != CUBEB_OK)
+          ERROR_LOG_FMT(AUDIO, "Error getting minimum latency");
+        INFO_LOG_FMT(AUDIO, "Minimum latency: {} frames", minimum_latency);
+
+        return_value = cubeb_stream_init(
+                         m_ctx.get(),
```

Comment:
Hmm. I don't care for this...

Reply:
Edit: I think this is our fault. An `ifdef` determines if this is a lambda body or not.